test/json_schema/reference_expander_test.rb

```diff
@@ -173,6 +174,12 @@
     # but diving deeper results in the same pointer again
     schema = schema.properties["app"]
     assert_equal "#/properties/app", schema.pointer
+
+    schema = @schema.properties["app1"]
+    assert_equal "#/properties/app1", schema.pointer
+
+    schema = schema.properties["app1"]
+    assert_equal "#/properties/app1", schema.pointer
```

Comment:
Sorry, my mistake. I thought both sets were using an instance variable `@schema`, but now see that's not the case.